MarkdownPicPicker.py

```diff
@@ -56,7 +56,7 @@ def init_environment(self):
             exit()
 
     def upload_picture(self):
-        picture_path_list = self.imageGrab.save_picture()
+        picture_path_list = [self.imageGrab.save_picture()]
```

Comment:
Do you develop in windows? If so, please correct the code in `ImageGrab/WinImageGrab.py` instead of here. In the line 21, the code should be:

```
return [picture_path]
```

instead of 

```
return picture_path
```

this is a bug, please help me fix it. I maintain this project mainly in macOS, and one day I develop the feature of copying multi files at one time for mac version, but didn't not update the code of windows version, so the bug occured.

Reply:
update done